.rustme/config.ron

```diff
@@ -42,256 +42,16 @@ Configuration(
                 default: "https://khonsulabs.github.io/nominals/main/nominals/",
```

Comment:
feel free to ignore this change, if you prefer the $Decimal$ in the `previews` output.

Reply:
I finally actually understand this change now. I like it.